specification/ai/Azure.AI.Unified/connections/models.tsp

```diff
@@ -0,0 +1,59 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@typespec/openapi";
+import "@typespec/versioning";
+import "../credentials/models.tsp";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Unified;
+
+@doc("Response from the listSecrets operation")
+@resource("connections")
+model Connection {
+  @doc("Fully qualified resource identifier")
+  @visibility("read")
+  id: string;
+
+  @doc("The name of the resource")
+  @visibility("read")
+  @key("connectionName")
+  name: string;
+
+  @doc("Credential used to connect to the external resource.")
+  @visibility("read")
+  credentials: BaseCredential;
```

Comment:
`credentials` should be optional. if you call "client.connection.get" it will return a Connection object without credentials.